shanoir-ng-datasets/src/main/resources/logback-spring.xml

```diff
@@ -58,7 +58,11 @@ along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
     <logger name="org.shanoir.ng.shared.event" level="INFO" additivity="false">
         <appender-ref ref="EVENT_FILE_LOG" />
     </logger>
-    
+	
+    <logger name="org.shanoir.ng.dataset.service.CreateStatisticsService" level="INFO" additivity="false">
+        <appender-ref ref="CREATE_STATS_LOG" />
```

Comment:
This <appender-ref> is supposed to point to a referenced appender in the file, here it is orpheline.
I think you should just put "FILE" here, so that it is set in the default shanoir-ng-datasets.log file.